syntax = "proto3";

package map_pb;

message RGB {
	int32 red   = 1;
	int32 green = 2;
	int32 blue  = 3;
}

message RGBO {
	int32  red     = 1;
	int32  green   = 2;
	int32  blue    = 3;
	double opacity = 4;
}

message ColorVar {
	string color = 1;
	RGB    rgb   = 2;
	RGBO   rgbo  = 3;
}

message Point {
	double x = 1;
	double y = 2;
}